src/Pie.js

```diff
@@ -18,37 +18,71 @@ function createPath(cx, cy, r, startAngle, arcAngle) {
       r,
       startAngle,
       startAngle + arcAngle,
-    )
+    );
   } else {
-    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle))
-    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1)
+    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle));
+    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1);
   }
-  return p
+  return p;
 }
 
-const ArcShape = ({ radius, width, color, strokeCap,startAngle, arcAngle }) => {
+const ArcShape = ({ radius, width, color, strokeCap, startAngle, arcAngle }) => {
   const path = createPath(
     radius,
     radius,
     radius - width / 2,
     startAngle / 180 * Math.PI,
-    arcAngle / 180 * Math.PI
-  )
-  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />
-}
+    arcAngle / 180 * Math.PI,
+  );
+  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />;
+};
 
-const RingShape = props =>
-  Platform.OS === 'ios'
-    ? <ArcShape {...props} startAngle={0} arcAngle={360} />
-    : <Group>
-        <ArcShape {...props} startAngle={0} arcAngle={180} />
-        <ArcShape {...props} startAngle={180} arcAngle={180} />
-      </Group>
+const RoundDividers = ({ paintedSections, dividerSize, width, radius, backgroundColor }) => {
+  let dividerColorOverlayArray = [];
+  let dividerArray = [];
+  paintedSections.forEach((section, index) => {
+    const { percentage, color, startAngle } = section;
+    const shouldShow = percentage !== 100;
```

Comment:
We are not rendering anything when this is true, hence we can include this to `shouldShowDivider` variable in main function. Comment Below

Reply:
what happens if `percentage > 100` or `percentage + dividerSize > 100`?

Reply:
100% is nothing but 360 arc, which is the maximum value that can be rendered. 

We will have to show some message when user tries to render pie with % > 100

Reply:
I agree that the use case of more than 100% makes no sense, maybe we can have a future PR that guards against for that. For now that's out of scope for this PR

Reply:
@nihgwu I will remove this for now because there's more work that would need to be done for this to be a useful addition

Reply:
OK, I can stand with the current solution